### PR TITLE
test: adds more implementations for pw+axe testing

### PR DESCRIPTION
**Pull request by soykje**
**TASK**: #2351 

### Description, Motivation and Context
Adds more implementations to test accessibility with PW+Axe workflow :muscle: 

### Types of changes
- [x] 🧪 Test

### CODE REVIEW

File: e2e/a11y/pages/FormField.tsx [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+export const A11yFormField = () => (
+  <section>
+    <div>
+      <FormField name="email">
+        <FormField.Label>Email</FormField.Label>
+
+        <FormField.Control>
+          {({ id, name, description }) => (
+            <input
```
Comment (acd02) [inline e2e/a11y/pages/FormField.tsx:12]: you should use a `@spark-ui/input` here

File: e2e/a11y/pages/FormField.tsx [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+      </FormField>
+    </div>
+
+    <div>
+      <FormField name="password" isRequired>
+        <FormField.Label>Password</FormField.Label>
+
+        <FormField.Control>
+          {({ id, name, description, isRequired }) => (
+            <input
```
Comment (acd02) [inline e2e/a11y/pages/FormField.tsx:32]: same here

File: e2e/a11y/pages/FormField.tsx (hunk 1 above)
Comment (soykje) [reply e2e/a11y/pages/FormField.tsx:12]: Why? This test should focus on form-field, shouldn't it?

Comment (acd02) [reply e2e/a11y/pages/FormField.tsx:12]: Yes, but why not make sure the `Formfield` works well with our input at the same time?

Comment (soykje) [reply e2e/a11y/pages/FormField.tsx:12]: Tbh I'm not sure... In this case shouldn't we add all our input components within Formfield implementation? :thinking:

**Review by acd02 — APPROVED** [review]

**Review by Powerplex — APPROVED** [review]

**Pull request merged**